### PR TITLE
Mention x and youtube

### PR DESCRIPTION
**Pull request by rickyzhangca**
before 

image

after 

image

and fixed a couple svg errors

### CODE REVIEW

File: settings/global.mdx [hunk 1]
```diff
@@ -249,7 +249,7 @@ Example:
   ```
   <Expandable title="FooterSocials">
     <ResponseField name="[key]" type="string">
-    One of the following values `website`, `facebook`, `twitter`, `discord`, `slack`, `github`, `linkedin`, `instagram`, `hacker-news`
+    One of the following values `website`, `facebook`, `twitter`, `x`, `youtube`, `discord`, `slack`, `github`, `linkedin`, `instagram`, `hacker-news`
     
     Example: `twitter`
```
Comment (handotdev) [inline settings/global.mdx]: Let's change the example here cause `twitter` is not the preferred value. Maybe `linkedin`?

File: settings/global.mdx [hunk 2]
```diff
@@ -249,7 +249,7 @@ Example:
   ```
   <Expandable title="FooterSocials">
     <ResponseField name="[key]" type="string">
-    One of the following values `website`, `facebook`, `twitter`, `discord`, `slack`, `github`, `linkedin`, `instagram`, `hacker-news`
+    One of the following values `website`, `facebook`, `twitter`, `x`, `youtube`, `discord`, `slack`, `github`, `linkedin`, `instagram`, `hacker-news`
```
Comment (handotdev) [inline settings/global.mdx]: Let's also removed `twitter` here. It can be allowed for backwards compatibility, but not suggested

**Review by handotdev — APPROVED** [review]
Oh also recommend updating the `mintlify/starter` repo!

**Review by handotdev — CHANGES_REQUESTED** [review]
Oh and also our own docs LMAOOO! Let's change our own footerSocial to `x` instead of `twitter`

**Comment by rickyzhangca** [comment]
oh haha sure time has changed

File: settings/global.mdx (hunk 1 above)
Comment (rickyzhangca) [reply settings/global.mdx]: do we want `"x": "https://twitter.com/mintlify"` @handotdev ?

**Review by handotdev — APPROVED** [review]

**Pull request merged**